codeworld-server/src/Main.hs

```diff
@@ -152,7 +152,8 @@ site ctx =
             [ ("loadProject", loadProjectHandler ctx)
             , ("saveProject", saveProjectHandler ctx)
             , ("deleteProject", deleteProjectHandler ctx)
-            , ("listFolder", listFolderHandler ctx)
+            , ("directoryTree", directoryTreeHandler ctx)
+            , ("dumpTree", dumpTreeHandler ctx)
```

Comment:
Whether you change the structure of the file or not, I'd really like to see this endpoint named something like `writeProjectOrder`.  To me, `dumpTree` sounds like a debugging endpoint, and it was really not obvious what this is for.

Reply:
If we going to store order inside of `dir.info`, this endpoint will have different logic or even not required.
If it will be required after I'll rewrite this solution, I'll rename it into `writeProjectOrder`.